tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -105,16 +111,20 @@ function install_artifacts() {
 	[ -d /opt/kata/runtime-rs/bin ] && \
 		chmod +x /opt/kata/runtime-rs/bin/*
 
+	config_path="/opt/kata/share/defaults/kata-containers/"
 	# Allow enabling debug for Kata Containers
 	if [[ "${DEBUG}" == "true" ]]; then
-		config_path="/opt/kata/share/defaults/kata-containers/"
 		for shim in "${shims[@]}"; do
 			sed -i -e 's/^#\(enable_debug\).*=.*$/\1 = true/g' "${config_path}/configuration-${shim}.toml"
 			sed -i -e 's/^#\(debug_console_enabled\).*=.*$/\1 = true/g' "${config_path}/configuration-${shim}.toml"
 			sed -i -e 's/^kernel_params = "\(.*\)"/kernel_params = "\1 agent.log=debug initcall_debug"/g' "${config_path}/configuration-${shim}.toml"
 		done
 	fi
 
+	if [ -n "${allowed_hypervisor_annotations}" ]; then
+		sed -i -e "s/^enable_annotations = \[\(.*\)\]/enable_annotations = [\1, $allowed_hypervisor_annotations]/" "${config_path}/configuration-${shim}.toml"
+	fi
```

Comment:
Maybe a dumb question, but will `shim` be set here, or should this be running inside the `for shim in "${shims[@]}"; do` loop?